services/galley/src/Galley/Data.hs

```diff
@@ -73,13 +72,14 @@ import Control.Applicative
 import Control.Arrow (second)
 import Control.Concurrent.Async.Lifted.Safe
 import Control.Lens hiding ((<|))
-import Control.Monad (join)
+import Control.Monad (join, forM)
 import Control.Monad.IO.Class
 import Control.Monad.Trans.Control
 import Data.ByteString.Conversion hiding (parser)
 import Data.Foldable (toList, foldrM, for_)
 import Data.Id
 import Data.Range
+import Data.List.Extra (chunksOf)
```

Comment:
The `chunksOf` from Extra is [partial](http://hackage.haskell.org/package/extra-1.6.2/docs/src/Data-List-Extra.html#chunksOf). Use the one from `Data.List.Split` instead?

Reply:
Oh nice! We use the version from `Extra` quite a bit, worth reviewing all those usages